libraries/k8s_namespace.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+require 'k8sobject'
+
+module Inspec
+  module Resources
+    # k8s_namespace resource to get data about specific kubernetes namespaces.
+    class K8sNamespace < K8sObject
+      DEFAULT_NAMESPACE = nil
+      name 'k8s_namespace'
+      desc 'Verifies settings for a specific namespace'
+
+      example "
+      describe k8s_namespace(name: 'kube-node-lease') do
+        it { should exist }
+        its('uid') { should eq '5ed76d62-838b-45cb-b41f-789b567a2fa2' }
+        its('name') { should eq 'kube-node-lease' }
+        its('kind') { should eq 'Namespace' }
+        its('resource_version') { should eq '6' }
+        its('creationTimestamp') { should eq '2022-07-21T10:47:49Z' }
```

Comment:
same here you aliased the` creationTimestamp` so why not use creation_timestamp?

Reply:
I aliased the `creationTimestamp` to `creation_timestamp` in the PR #15 (CFINSPEC-320: Add Kubernetes resources: k8s_deployment and k8s_deployments) and the branch from which the PR is raised is checked out from main.

The branch from which the current PR i.e. PR #16 is raised is also checked out from main. I could have either re-introduced the alias in this branch which could have created a conflict while merging or wait for the other PR to be merged to use `creation_timestamp` in this PR.

Imo the best option was to document the alias and let the user use either `creationTimestamp` or `creation_timestamp`